crux-mir-comp/DESIGN.md

```diff
@@ -147,6 +147,55 @@ implementations of some cryptographic algorithms) are usually declared as
 immutable and thus always retain their initial values.
 
 
+## Cryptol
+
+It is possible to import Cryptol expressions into symbolic Rust test cases to
+help write specifications (e.g., if you want to check that a Rust implementation
+matches a Cryptol specifications).  This is done by using the `cryptol!` macro
+defined in the `crucible` package.  Here's an example illustrating how to
+import a Cryptol function `myCryFun` defined in Cryptol module `SomeCryMod`,
+and bind it as the Rust function `myRustFun`:
+
+```Rust
+use crucible::*;
+cryptol! {
+    path "SomeCryMod";
+    pub fn myRustFun(x: u8) -> u32 = r"myCryFun";
```

Comment:
It would also be worth showing the code in `SomeCryMod.cry` to drive home the idea that the Cryptol types have to match up with the Rust types.